Gemfile

```diff
@@ -27,6 +27,7 @@ gem 'rubocop', '>= 1.0', '< 2.0'
 
 group :development, :test do
   gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'rspec-rails'
 end
 
```

Comment:
- [OPTIONAL] Since I'm not able to inline comment on your license I just do it here but I noticed that the link provided is not working properly, Kindly fix this for your project to match all requirements for the project :wink: 
![Screenshot from 2023-02-25 20-34-15](https://user-images.githubusercontent.com/88894525/221374067-79ac956a-63cb-4bde-8097-203bfdf57447.png)